Fix format and add important note about CAS in baseline-aks-content.md (#4490)
MicrosoftDocs/architecture-center: docs/reference-architectures/containers/aks/baseline-aks-content.md

- Enable the cluster autoscaler to detect and remove underutilized nodes in the node pool.

> [!IMPORTANT]

> > Proactively changing cluster autoscaler settings (such as min/max node settings for a node pool) for cost reasons will have counterproductive results. For example, if `scale-down-unneeded-time` is set to 10 minutes, and the min/max node settings are modified every 5 minutes based on workload, the number of nodes will never go down. This is because the calculation of the unneeded time for each node will be reset due to the cluster autoscaler settings being refreshed. 

Reviewer: ```suggestion
> Aggressively changing cluster autoscaler settings (such as min/max node settings for a node pool) for cost reasons might have counterproductive results. For example, if `scale-down-unneeded-time` is set to 10 minutes, and the min/max node settings are modified every five minutes based on workload characteristics, the number of nodes will never go down. This is because the calculation of the unneeded time for each node will be reset due to the cluster autoscaler settings being refreshed.
```

> > 
> > Therefore, if for FinOps consideration, it is indeed necessary to proactively change the cluster autoscaler settings, consider the consequences of the scale-down process being delayed.

Reviewer: ```suggestion
```

- Enable the cluster autoscaler to detect and remove underutilized nodes in the node pool.

> > [!IMPORTANT]
> > Aggressively changing cluster autoscaler settings (such as min/max node settings for a node pool) for cost reasons might have counterproductive results. For example, if `scale-down-unneeded-time` is set to 10 minutes, and the min/max node settings are modified every five minutes based on workload characteristics, the number of nodes will never go down. This is because the calculation of the unneeded time for each node will be reset due to the cluster autoscaler settings being refreshed.

Reviewer: ```suggestion
  > [!IMPORTANT]
  > Aggressively changing cluster autoscaler settings (such as min/max node settings for a node pool) for cost reasons might have counterproductive results. For example, if `scale-down-unneeded-time` is set to 10 minutes, and the min/max node settings are modified every five minutes based on workload characteristics, the number of nodes will never go down. This is because the calculation of the unneeded time for each node will be reset due to the cluster autoscaler settings being refreshed.
```
